doc/.wordlist.txt

```diff
@@ -25,6 +25,15 @@ Kubeflow
 Kubernetes
 Launchpad
 linter
+kB
+kbit
+KiB
+kibi
+Kibit
+KVM
+lookups
+Loongarch
+LRU
```

Comment:
Most of these terms aren't needed. Unless I missed any, the only one used in this PR is "Loongarch".
Also, we don't edit `.wordlist.txt` but instead add exceptions to `.custom_wordlist.txt` (see the comment at the top of the file).